packages/foundation-legacy/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "noImplicitAny": true,
     "moduleResolution": "node",
     "preserveConstEnums": true,
-    "lib": ["es5", "dom"],
+    "lib": ["es5", "dom", "ES2015.Iterable"],
```

Comment:
`@types/react@17` references `Iterable` so this had to be done here and in other v8 packages that target es5